msgid ""
msgstr ""
"Project-Id-Version: BackupSuite\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-07-22 19:19+0200\n"
"PO-Revision-Date: \n"
"Last-Translator: Pedro_Newbie <backupsuite@outlook.com>\n"
"Language-Team: Vytenis P. <vyteniui@gmail.com>\n"
"Language: lt\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Generator: Poedit 2.1.1\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && (n%100<10 || n%100>=20) ? 1 : 2);\n"

#: ../plugin.py:441
msgid ""
"\n"
"The found files:"
msgstr ""
"\n"
"Surasti failai:"

#: ../plugin.py:358
msgid ""
"\n"
"You are using dual multiboot!"
msgstr ""

#: ../plugin.py:447
#, python-format
msgid "  %s (maybe error)"
msgstr "  %s (galbūt klaida)"

#: ../plugin.py:444
#, python-format
msgid "  %s (maybe ok)"
msgstr "  %s (galbūt gerai)"

#: ../message.py:180
msgid " MB available space"
msgstr " MB laisvos vietos"

#: ../message.py:185
msgid " MB needed space"
msgstr " MB reikia"

#: ../plugin.py:450
msgid " nothing!"
msgstr " nieko!"

#: ../message.py:45
msgid " size to be backed up: "
msgstr " atsarginės kopijos dydis: "

#: ../message.py:175
msgid " to make a back-up!"
msgstr " kad padaryti atsarginę kopiją!"

#: ../message.py:169
msgid "* * * WARNING * * *"
msgstr "* * * DĖMESIO * * *"

#: ../plugin.py:480
msgid "A recording is currently running. Please stop the recording before trying to start a flashing."
msgstr "Vyksta įrašymas. Prieš atvaizdo įdiegimą, nutraukite įrašymą."

#: ../message.py:70
msgid "Additional backup -> "
msgstr "Papildoma kopija -> "

#: ../message.py:35
msgid "BACK-UP TOOL, FOR MAKING A COMPLETE BACK-UP"
msgstr "ATSARGINIO KOPIJAVIMO ĮRANKIS PILNOMS KOPIJOMS KURTI"

#: ../plugin.py:132
msgid "Backup > HDD"
msgstr "Kopijuoti > HDD"

#: ../plugin.py:130
#, fuzzy
msgid "Backup > MMC"
msgstr "Kopijuoti > HDD"

#: ../plugin.py:133
msgid "Backup > USB"
msgstr "Kopijuoti > USB"

#: ../plugin.py:584 ../plugin.py:591
msgid "Backup and restore your image"
msgstr ""

#: ../message.py:149
msgid "Backup done with: "
msgstr "Atsarginė kopija padaryta: "

#: ../message.py:106
msgid "Backup finished and copied to your USB-flashdrive."
msgstr "Atsarginis kopijavimas baigtas, kopija įkelta į USB laikmeną."

#: ../plugin.py:338 ../plugin.py:541
#, fuzzy
msgid "Backup info"
msgstr "BackupSuite"

#: ../plugin.py:583 ../plugin.py:590
msgid "BackupSuite"
msgstr "BackupSuite"

#: ../plugin.py:131 ../plugin.py:233 ../plugin.py:262
msgid "Close"
msgstr ""

#: ../message.py:65
msgid "Create: kerneldump"
msgstr "Sukurti: kerneldump"

#: ../message.py:60
msgid "Create: root.ubifs"
msgstr "Sukurti: root.ubifs"

#: ../plugin.py:339 ../plugin.py:570
msgid "Delete"
msgstr ""

#: ../plugin.py:567
#, fuzzy
msgid "Delete backup"
msgstr "Atkurti atsarginę kopiją"

#: ../plugin.py:540
#, python-format
msgid "Do you really want to unpack %s ?"
msgstr "Ar tikrai norite išskleisti %s?"

#: ../plugin.py:153
msgid ""
"Do you want to make a back-up on USB?\n"
"\n"
"This only takes a few minutes depending on the used filesystem and is fully automatic.\n"
"\n"
"Make sure you first insert an USB flash drive before you select Yes."
msgstr ""
"Jūs norite padaryti atvaizdo atsarginę kopiją į USB laikmeną?\n"
"\n"
"Tai užtrunka tik keletą minučių, priklausomai nuo failų sistemos ir viskas atliekama automatiškai.\n"
"\n"
"Prieš pasirinkdami 'Taip', įsitikinkite, kad prijungėte USB laikmeną."

#: ../plugin.py:150
msgid ""
"Do you want to make an USB-back-up image on HDD? \n"
"\n"
"This only takes a few minutes and is fully automatic.\n"
msgstr ""
"Jūs norite padaryti atvaizdo atsarginę USB įkrovos kopiją į HDD?\n"
"\n"
"Tai užtrunka tik keletą minučių ir viskas atliekama automatiškai.\n"

#: ../plugin.py:156
#, fuzzy
msgid ""
"Do you want to make an USB-back-up image on MMC? \n"
"\n"
"This only takes a few minutes and is fully automatic.\n"
msgstr ""
"Jūs norite padaryti atvaizdo atsarginę USB įkrovos kopiją į HDD?\n"
"\n"
"Tai užtrunka tik keletą minučių ir viskas atliekama automatiškai.\n"

#: ../plugin.py:470
msgid "Exit"
msgstr "Užverti"

#: ../plugin.py:484
msgid "Flashing: "
msgstr "Diegiama: "

#: ../plugin.py:369
msgid "For flashing your receiver files are needed:\n"
msgstr "Įdiegimui reikalingi failai:\n"

#: ../message.py:129
msgid "Full back-up direct to USB"
msgstr "Pilna atsarginė kopija tiesiai į USB laikmeną"

#: ../plugin.py:194
msgid "Full back-up on HDD"
msgstr "Pilna atsarginė kopija į HDD"

#: ../plugin.py:208
#, fuzzy
msgid "Full back-up on MMC"
msgstr "Pilna atsarginė kopija į HDD"

#: ../plugin.py:201
msgid "Full back-up to USB"
msgstr "Pilna atsarginė kopija į USB"

#: ../message.py:235
#, fuzzy
msgid "Full back-up to the MultiMediaCard"
msgstr "Pilna atsarginė kopija į HDD"

#: ../message.py:111
msgid "Full back-up to the harddisk"
msgstr "Pilna atsarginė kopija į HDD"

#: ../message.py:123
msgid "If you place an USB-stick containing this file then the back-up will be automatically made onto the USB-stick and can be used to restore the current image if necessary."
msgstr "Jei prijungsite USB laikmeną su šiuo failu, į ją bus automatiškai sukurta atsarginė kopija ir prireikus, ši laikmena gali būti naudojama atkurti dabartinį atvaizdą."

#: ../message.py:90
msgid "Image creation FAILED!"
msgstr "Atvaizdo sukurti NEPAVYKO!"

#: ../message.py:225
msgid "Installed packages contained in this backup:"
msgstr ""

#: ../message.py:154
msgid "KB per second"
msgstr "KB per sekundę"

#: ../plugin.py:127
msgid "Make a backup or restore a backup"
msgstr "Padaryti, arba atkurti atsarginę kopiją"

#: ../message.py:159
msgid "Most likely this back-up can't be restored because of it's size, it's simply too big to restore. This is a limitation of the bootloader not of the back-up or the BackupSuite."
msgstr "Greičiausiai šios atsarginės kopijos nebegalima atkurti dėl jos dydžio - ji tiesiog per didelė, kad atkurti. Tai ne atvaizdo ir ne šio kopijavimo įrankio, bet OS paleidimo programos apribojimas."

#: ../plugin.py:45
msgid "NAS"
msgstr ""

#: ../message.py:230
msgid "NFI Image created in: "
msgstr ""

#: ../plugin.py:516
msgid "NOT found files for flashing!\n"
msgstr "Failų diegimui NERASTA!\n"

#: ../message.py:220
msgid "No additional USB-stick found to copy an extra backup"
msgstr ""

#: ../plugin.py:173 ../message.py:30
msgid "No supported receiver found!"
msgstr "Nerasta tinkamo imtuvo!"

#: ../message.py:170
msgid "Not enough free space on "
msgstr "Nepakanka laisvos vietos "

#: ../plugin.py:458 ../plugin.py:499
msgid "Only kernel"
msgstr "Tik branduolį"

#: ../plugin.py:457 ../plugin.py:496
msgid "Only root"
msgstr "Tik pagrindinį"

#: ../message.py:118
msgid "PLEASE READ THIS:"
msgstr "PERSKAITYKITE :"

#: ../message.py:40
msgid "Please be patient, a backup will now be made, this will take about: "
msgstr "Būkite kantrūs, atsarginė kopija tuojau bus padaryta. Tai užtruks maždaug: "

#: ../message.py:85
msgid "Please check the manual of the receiver on how to restore the image."
msgstr "Kaip atkurti sistemos atvaizdą, žiūrėkite savo aparato vartotojo vadove."

#: ../plugin.py:234
msgid "Please wait"
msgstr ""

#: ../plugin.py:528
msgid "Please: DO NOT reboot your STB and turn off the power.\n"
msgstr "NEPALEIDINĖKITE aparato iš naujo ir NEIŠJUNKITE maitinimo įtampos.\n"

#: ../message.py:215
msgid "Probably you are trying to make the back-up in flash memory"
msgstr ""

#: ../plugin.py:134
msgid "Restore backup"
msgstr "Atkurti atsarginę kopiją"

#: ../plugin.py:336 ../plugin.py:353 ../plugin.py:362
msgid "Run flash"
msgstr "Pradėti diegimą"

#: ../plugin.py:464 ../plugin.py:511
#, fuzzy
msgid "Second partition (only kernel)"
msgstr "Standartinis (atvaizdas ir branduolys)"

#: ../plugin.py:463 ../plugin.py:508
msgid "Second partition (only root)"
msgstr ""

#: ../plugin.py:462 ../plugin.py:505
#, fuzzy
msgid "Second partition (root and kernel)"
msgstr "Standartinis (atvaizdas ir branduolys)"

#: ../plugin.py:368
msgid "Select parameter for start flash!\n"
msgstr "Pasirinkite diegimo parametrus!\n"

#: ../plugin.py:261
msgid "Select the folder with backup"
msgstr "Pasirinkite katalogą su atsargine kopija"

#: ../plugin.py:521
msgid "Show only found image and mtd partitions.\n"
msgstr "Rodyti tik rastą atvaizdą ir MTD skaidinius.\n"

#: ../plugin.py:455 ../plugin.py:487
msgid "Simulate (no write)"
msgstr "Imituoti (be įrašymo)"

#: ../plugin.py:461 ../plugin.py:502
#, fuzzy
msgid "Simulate second partition (no write)"
msgstr "Imituoti (be įrašymo)"

#: ../message.py:55
msgid "Some information about the task"
msgstr "Šiek tiek informacijos apie užduotį"

#: ../plugin.py:456 ../plugin.py:490
msgid "Standard (root and kernel)"
msgstr "Standartinis (atvaizdas ir branduolys)"

#: ../message.py:134
msgid "The content of the folder is:"
msgstr "Katalogo turinys:"

#: ../plugin.py:529
msgid "The image or kernel will be flashing and auto booted in few minutes.\n"
msgstr "Atvaizdas, arba branduolys bus įdiegtas ir automatiškai pasileis po kelių minučių.\n"

#: ../message.py:190
msgid "The program will abort, please try another medium with more free space to create your back-up."
msgstr "Veiksmas bus nutrauktas, atsarginei kopijai pasirinkite kitą laikmeną su daugiau laisvos vietos."

#: ../message.py:124
msgid "The program will exit now."
msgstr "Programa dabar užsidarys."

#: ../message.py:164
msgid "There COULD be a problem with restoring this back-up because the size of the back-up comes close to the maximum size. This is a limitation of the bootloader not of the back-up or the BackupSuite."
msgstr "GALI būti problemų su šios atsarginės kopijos atkūrimu, nes jos dydis jau priartėja prie maksimalaus leistino. Tai ne atvaizdo ir ne šio kopijavimo įrankio, bet OS paleidimo programos apribojimas."

#: ../message.py:116
msgid "There is NO valid USB-stick found, so I've got nothing to do."
msgstr "Tinkamų USB laikmenų nerasta, todėl nėra ką daryti."

#: ../message.py:100
msgid "There is a valid USB-flashdrive detected in one of the USB-ports, therefore an extra copy of the back-up image will now be copied to that USB-flashdrive."
msgstr "Viename iš USB prievadų aptikta tinkama USB laikmena, todėl papildoma atsarginė atvaizdo kopija bus nukopijuota į tą USB laikmeną."

#: ../message.py:101
msgid "This only takes about 20 seconds....."
msgstr "Tai truks tik apie 20 sekundžių...."

#: ../message.py:139
msgid "Time required for this process: "
msgstr "Šis procesas užtruko: "

#: ../message.py:119
msgid "To back-up directly to the USB-stick, the USB-stick MUST contain a file with the name:"
msgstr "Kad atsarginę kopiją daryti tiesiai į USB atmintinę, USB laikmenoje TURI BŪTI failas pavadinimu:"

#: ../message.py:75
msgid "USB Image created in: "
msgstr "USB atvaizdas sukurtas: "

#: ../plugin.py:334 ../plugin.py:537
msgid "Unzip"
msgstr "Išskleisti"

#: ../message.py:200
msgid "Using your own custom lookuptable.txt from the folder /etc"
msgstr ""

#: ../message.py:205
msgid "Version unknown, probably not installed the right way."
msgstr ""

#: ../plugin.py:359
msgid ""
"Warning!\n"
"Use at your own risk! Make always a backup before use!\n"
"Don't use it if you use multiple ubi volumes in ubi layer!"
msgstr ""
"Įspėjimas!\n"
"Naudokite savo atsakomybe! Prieš naudojimą, visada padarykite atsarginę kopiją!\n"
"Nenaudokite jo, jei naudojate  keliatomius UBI!"

#: ../plugin.py:232
msgid "What is new since the last release?"
msgstr "Kas naujo nuo paskutinio išleidimo?"

#: ../plugin.py:571
#, python-format
msgid ""
"You are about to delete this backup:\n"
"\n"
"%s\n"
"Continue?"
msgstr ""

#: ../plugin.py:527
msgid "Your STB will freeze during the flashing process.\n"
msgstr "Diegimo metu jūsų imtuvas nustos reaguoti į komandas.\n"

#: ../message.py:80
msgid "and there is made an extra copy in: "
msgstr "ir padaryta papildoma kopija į: "

#: ../message.py:95
msgid "available "
msgstr "prieinama "

#: ../message.py:120
msgid "backupstick or"
msgstr "backupstick, arba"

#: ../message.py:121
msgid "backupstick.txt"
msgstr "backupstick.txt"

#: ../plugin.py:266 ../plugin.py:328
#, python-format
msgid "current:  %s"
msgstr "dabar: %s"

#: ../message.py:195
msgid "is not executable..."
msgstr ""

#: ../message.py:144
msgid "minutes"
msgstr "min."

#: ../message.py:50
msgid "not found, the backup process will be aborted!"
msgstr "nerasta, atsarginio kopijavimo procesas bus nutrauktas!"

#: ../message.py:210
msgid "not installed yet, now installing"
msgstr ""

#: ../plugin.py:526
msgid "ofgwrite will stop enigma2 now to run the flash.\n"
msgstr "ofgwrite dabar sustabdys enigma2 ir pradės diegimą.\n"
